frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.tsx

```diff
@@ -20,25 +20,22 @@ import type { IInternalLayout } from '@altinn/ux-editor/types/global';
 import { PdfConfig } from '@altinn/ux-editor/components/Properties/PageConfigPanel/PdfConfig';
 
 export const PageConfigPanel = () => {
-  const { selectedFormLayoutName } = useAppContext();
+  const { selectedItem } = useAppContext();
+
   const t = useText();
   const modalRef = useRef<HTMLDialogElement>(null);
-  const layoutIsSelected =
-    selectedFormLayoutName !== DEFAULT_SELECTED_LAYOUT_NAME && selectedFormLayoutName !== undefined;
   const layoutNameTextResourceSelector = textResourceByLanguageAndIdSelector(
     DEFAULT_LANGUAGE,
-    selectedFormLayoutName,
+    selectedItem?.id,
   );
   const layoutNameTextResource = useTextResourcesSelector<ITextResource>(
     layoutNameTextResourceSelector,
   );
   const layoutNameText = layoutNameTextResource?.value;
-  const headingTitle = !layoutIsSelected
-    ? t('right_menu.content_empty')
-    : (layoutNameText ?? selectedFormLayoutName);
+  const headingTitle = layoutNameText || selectedItem?.id || t('right_menu.content_empty');
```

Comment:
```suggestion
  const headingTitle = layoutNameText || selectedItem.id
```

id will always be present, see comment above.